app/scripts/templates/sign_in.mustache

```diff
@@ -19,9 +19,13 @@
     {{^error}}
       <div class="error"></div>
       <div class="success"></div>
-        {{#isMigration}}
+      {{#isSyncMigration}}
           <div class="info nudge">{{#t}}Migrate your sync data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
-        {{/isMigration}}
+      {{/isSyncMigration}}
+
+      {{#isAmoMigration}}
+          <div class="info nudge">{{#t}}Migrate your Mozilla Add-ons data by signing in to your Firefox&nbsp;Account.{{/t}}</div>
```

Comment:
Sounds good, will remove. I think I just misread to original comment that it had to show migration on both pages.